### PR TITLE
Adds a new field and function to gather an impact statement from a post

### PR DESCRIPTION
**Pull request by lkpttn**
Adds pluralize package and runs the quantity + noun through it to make a cohesive impact statement.

https://www.pivotaltracker.com/story/show/169045990

### CODE REVIEW

**Comment by mendelB** [comment]
Whaaaaaat hey @lkpttn 👋🔥🔥🔥

File: src/repositories/rogue.js [hunk 1]
```diff
@@ -212,6 +213,19 @@ export const toggleReaction = async (postId, context) => {
...
 
+/**
+ * Create an impact statement from quantity, noun and verb
+ *
+ * @param {Object} post
+ * @return {String}
+ */
+export const makeImpactStatement = post => {
+  const { noun, verb } = post.actionDetails.data;
+  const statement = pluralize(noun, post.quantity, true);
```
Comment (katiecrane) [inline src/repositories/rogue.js]: I want to know more about this `pluralize` function! It seems really cool, what does it do?

Comment (DFurnes) [reply src/repositories/rogue.js]: Yeah it's very cool! It's an [npm package](https://www.npmjs.com/package/pluralize) that takes a noun and pluralizes (or singularizes) it based on the number provided, so we don't have to have both a `singularNoun` and `pluralNoun` that admins enter.

Comment (katiecrane) [reply src/repositories/rogue.js]: THAT IS SO COOL

**Pull request merged**